layers/+lang/c-c++/funcs.el

```diff
@@ -165,7 +165,9 @@
 
 (defun spacemacs//c-c++-setup-lsp-dap ()
   "Setup DAP integration."
-  (require 'dap-gdb-lldb))
+  (mapcar (lambda (dap-mode)
```

Comment:
`dap-mode` is a minor mode (and also a buffer-local variable).
To avoid any possible confusion and increase readability, I suggest to rename the argument, for example just name it `x`.
In fact, the entire lambda can be written in a point-free style so you don't need to worry about the naming.

Also `mapc` is more appropriate here because we only want the side-effect.

```elisp
(mapc #'require c-c++-dap-adapters)
```

Reply:
Done. Thanks!